client/src/components/GridFieldActions/GridFieldActions.js

```diff
@@ -6,21 +6,6 @@ import classnames from 'classnames';
 import PropTypes from 'prop-types';
 
 class GridFieldActions extends PureComponent {
-  constructor(props) {
-    super(props);
-
-    this.toggle = this.toggle.bind(this);
-    this.state = {
-      dropdownOpen: false
-    };
-  }
-
-  toggle() {
-    this.setState({
-      dropdownOpen: !this.state.dropdownOpen
-    });
-  }
```

Comment:
The `toggle` method and `dropdownOpen` state were never being referenced anywhere so I've removed them.